ui/src/app/pages/signed-in/component.ts

```diff
@@ -84,21 +74,12 @@ export class SignedInComponent implements OnInit, OnDestroy, AfterViewInit {
   ngOnInit(): void {
     this.serverConfigService.getConfig().subscribe((config) => {
       this.profileLoadingSub = this.profileStorageService.profile$.subscribe((profile) => {
-        this.hasDataAccess = hasRegisteredAccess(profile.dataAccessLevel);
-        if (this.hasDataAccess) {
+        this.profile = profile as unknown as FetchProfile;
+        if (hasRegisteredAccessFetch(this.profile.dataAccessLevel)) {
```

Comment:
I know you didn't name this in this PR but this function name is confusing and I'm wondering if there's a better name for it

Reply:
I think eventually it will be `hasRegisteredAccess`, its just this way until we can get rid of the angular version. (Maybe we should rename the angular one and make this `hasRegisteredAccess`?)